swarm/network/stream/syncer.go

```diff
@@ -127,19 +127,9 @@ func (s *SwarmSyncerServer) SetNextBatch(from, to uint64) ([]byte, uint64, uint6
 
 // SwarmSyncerClient
 type SwarmSyncerClient struct {
-	sessionAt     uint64
```

Comment:
WOW - what's going on here? Why is there so much "obsolence" in this code? @zelig @janos